pkgs/by-name/ad/adguard-exporter/package.nix

```
@@ -0,0 +1,21 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule {
+  name = "adguard-exporter";
```

```suggestion
  pname = "adguard-exporter";
  version = "0-unstable-2020-11-07";
```

See https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming